src/backend/distributed/sql/udfs/citus_copy_shard_placement/latest.sql

```diff
@@ -1,19 +1,16 @@
-DROP FUNCTION pg_catalog.citus_copy_shard_placement;
+-- citus_copy_shard_placement, but with nodeid
 CREATE FUNCTION pg_catalog.citus_copy_shard_placement(
```

Comment:
This file should have both the nodeid and the hostname+port definitions of this function, since we'll keep both for the foreseeable future. You can use CREATE OR REPLACE FUNCTION to recreate the hostname+port version without the need to drop it.